app/models/admin_dashboard_data.rb

```diff
@@ -129,57 +120,6 @@ def self.load_found_scheduled_check_problems
     end
   end
 
-  def self.register_default_scheduled_problem_checks
-    add_scheduled_problem_check(:group_smtp_credentials, GroupEmailCredentialsCheck) do
-      problems = GroupEmailCredentialsCheck.run
-      problems.map do |p|
-        problem_message =
-          I18n.t(
-            "dashboard.group_email_credentials_warning",
-            {
-              base_path: Discourse.base_path,
-              group_name: p[:group_name],
-              group_full_name: p[:group_full_name],
-              error: p[:message],
-            },
-          )
-        Problem.new(
```

Comment:
In the new API, any implementation of `ProblemCheck` already returns an array of `Problem` objects.